README.md

```diff
@@ -232,6 +232,9 @@ To release a new version:
   * Check that [each merged
     PR](https://github.com/KissKissBankBank/kitten/commits/master)
     from the last release has an entry.
+- Update the `KARL_CHANGELOG.md` file:
+  * Update the version with the version of the library (this is temporary).
```

Comment:
Tant qu'on fait comme ça, je serais d'avis de retirer `(this is temporary)`. Ça y sera encore dans 5 ans sinon ;)